### PR TITLE
SEO updates

### PR DESCRIPTION
**Pull request by Jashpatel1**
**Description**

This PR fixes titles and metadata.

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,7 +17,8 @@
     "start": "npm run develop",
     "serve": "gatsby serve",
     "lint": "eslint ./src --fix",
-    "pretest": "eslint --ignore-path .gitignore ."
+    "pretest": "eslint --ignore-path .gitignore .",
+    "preload-fonts": "gatsby-preload-fonts"
```
Comment (Tanuj22) [inline package.json:21]: How is this script going to be used?

Comment (Jashpatel1) [reply package.json:21]: `npm run preload-fonts`

Comment (Tanuj22) [reply package.json:21]: So, use this before `npm run build`?

File: src/pages/learn.js [hunk 2]
```diff
@@ -20,7 +20,7 @@ const WorkshopsGridPage = () => {
         <ThemeProvider theme={theme}>
             <Layout>
                 <GlobalStyle />
-                <SEO title="Learn | Layer5 - The Service Mesh Company" />
```
Comment (Tanuj22) [inline src/pages/learn.js:23]: Can you do this for other pages too. (Blog is one of them)

**Review by Tanuj22 — COMMENTED** [review]
We will want to add a `meta` tag for each page also right?

**Comment by Jashpatel1** [comment]
> We will want to add a `meta` tag for each page also right?

Yes, we will have to discuss the description text as well.

File: package.json (hunk 1 above)
Comment (Tanuj22) [reply package.json:21]: @Jashpatel1 ^^

**Comment by Tanuj22** [comment]
> > We will want to add a `meta` tag for each page also right?
> 
> Yes, we will have to discuss the description text as well.



> > We will want to add a `meta` tag for each page also right?
> 
> Yes, we will have to discuss the description text as well.

Can't the already present discription be used

File: package.json (hunk 1 above)
Comment (Jashpatel1) [reply package.json:21]: It has to be run only when fonts are changed.

**Comment by Jashpatel1** [comment]
> > > We will want to add a `meta` tag for each page also right?
> > 
> > 
> > Yes, we will have to discuss the description text as well.
> 
> Can't the already present discription be used

@leecalcote any thoughts on these?

**Review by Tanuj22 — APPROVED** [review]
Description yet to added

File: src/pages/landscape.js [hunk 3]
```diff
@@ -12,7 +12,7 @@ const LandscapeGridPage=({ data }) => (
     <ThemeProvider theme={theme}>
         <Layout>
             <GlobalStyle />
-            <SEO title="Landscape | Layer5 - The Service Mesh Company" />
+            <SEO title="Landscape" />
```
Comment (leecalcote) [inline src/pages/landscape.js:15]: Service Mesh Landscape

**Review by leecalcote — COMMENTED** [review]
Yes, please do include descriptions for each page. There should be unique descriptions provided for each existing page.

**Pull request merged**